routes/feeds.js

```diff
@@ -66,7 +66,7 @@ router.post('/comment-child-v2', isAuth, CreateCommentChildMiddleware, feed.comm
 router.post('/comment-domain-v2', isAuth, CreateCommentMiddleware, feed.commentDomainV2);
 router.post(
   '/block-anonymous-post-v2',
-  isAuth,
+  isAuthV2,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The authentication middleware has been changed from `isAuth` to `isAuthV2`. Ensure that `isAuthV2` is thoroughly tested and provides the same level of security as `isAuth`. If `isAuthV2` introduces new behavior, make sure it doesn't break existing functionality or introduce security vulnerabilities.
```diff
-  isAuth,
+  isAuthV2,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->